src/cljam/io/bam_index/writer.clj

```diff
@@ -4,26 +4,27 @@
             [cljam.io.sam.util :as sam-util]
             [cljam.io.util.bgzf :as bgzf]
             [cljam.io.util.lsb :as lsb]
-            [cljam.io.bam.decoder :as bam-decoder]
             [cljam.io.bam-index.common :refer :all]
-            [cljam.io.bam-index.chunk :as chunk])
-  (:import [java.nio ByteBuffer ByteOrder]))
+            [cljam.io.bam-index.chunk :as chunk]
+            [cljam.io.bam.decoder])
```

Comment:
I noticed that the decoder was used to perform decoding in parallel for better performance.
I restored the decoding part.
Thank you.